orchagent/routeorch.cpp

```diff
@@ -1612,19 +1686,28 @@ bool RouteOrch::addRoute(RouteBulkContext& ctx, const NextHopGroupKey &nextHops)
             /* IP neighbor is not yet resolved */
             else
             {
-                if(overlay_nh)
+                if(overlay_nh && !srv6_nh)
                 {
-                    SWSS_LOG_INFO("create remote vtep %s", nexthop.to_string(overlay_nh).c_str());
+                    SWSS_LOG_INFO("create remote vtep %s", nexthop.to_string(overlay_nh, srv6_nh).c_str());
                     status = createRemoteVtep(vrf_id, nexthop);
                     if (status == false)
                     {
-                        SWSS_LOG_ERROR("Failed to create remote vtep %s", nexthop.to_string(overlay_nh).c_str());
+                        SWSS_LOG_ERROR("Failed to create remote vtep %s", nexthop.to_string(overlay_nh, srv6_nh).c_str());
                         return false;
                     }
                     next_hop_id = m_neighOrch->addTunnelNextHop(nexthop);
                     if (next_hop_id == SAI_NULL_OBJECT_ID)
                     {
-                        SWSS_LOG_ERROR("Failed to create Tunnel Nexthop %s", nexthop.to_string(overlay_nh).c_str());
+                        SWSS_LOG_ERROR("Failed to create Tunnel Nexthop %s", nexthop.to_string(overlay_nh, srv6_nh).c_str());
+                        return false;
+                    }
+                }
+                else if (srv6_nh)
+                {
+                    SWSS_LOG_NOTICE("Single NH: create srv6 nexthop %s", nextHops.to_string().c_str());
```

Comment:
Please change the log level to INFO. Suggest not to have debug type logs in route add path. Revisit other 'NOTICE' logs as well in the PR

Reply:
Changed NOTICE logs to _INFO